addons/fieldmanual/stringtable.xml

```diff
@@ -0,0 +1,359 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="FieldManual">
+        <Key ID="STR_ACE_FieldManual_Field_Rations_DisplayName">
+            <English>Field Rations</English>
```

Comment:
I'm not a fan of duplicating these entries, but they're not likely to change anyway.

Reply:
Do you mean duplicating hunger and thirst? I think in my original idea I was going to have the current ACE setting values be put into the strings (for the whole of this addon), but decided against it. Would you like me to remove and consolidate the hunger and thirst entries?

Reply:
What I mean is, they're already available in Field Ration's stringtable. But reusing them would create a dependency, so it's fine.

Reply:
Oh I see what you mean. Yeah I was debating using the strings from the addons themselves, but wanted to make it completely standalone for now. Can always change it later.

Reply:
Could just put the manual entries in each addon instead of having it add a single separate addon